draft-thomson-ppm-prss.md

```diff
@@ -186,7 +186,7 @@ def ss, enc = Send(kem, pk_bytes):
 ~~~
 
 The sender then sends the encapsulated public key, `enc`, to the receiver.  The
-receiver decapsulates this value to obtain the shared secret, `secret`:
+receiver decapsulates this value to obtain the shared secret, `ss`:
```

Comment:
`ss` seems to better match the identifier used in the pseudo-Python on L192-193 and the line on L196.